Application/Setting/Univention/Service/Entity/TblUnivention.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Codacy found an issue: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)